react/Modal/index.jsx

```diff
@@ -8,6 +8,11 @@ const ModalContent = ({children, className}) =>
     {children}
   </div>)
 
+const ModalContentOffset = ({children, className}) =>
```

Comment:
ModalContent is the basic one with paddings, used in most cases.
ModalContentOffset doesn't have paddings so we can do anything inside (like tabs that take the all with of the modal).